### DISCUSSION TITLE
[TTS] Cantonese sounds not right

### DISCUSSION THREAD

**Discussion by garywill** [PaddlePaddle/PaddleSpeech#3623] [General]
粤语使用例子中的方式生成，生成的声音每到断句处（逗号、句号、句末），最后一个字都会被吞掉

am使用`fastspeech2_canton`，lang设置`canton`。试过了不同的voc和spk_id都一样

> 一二三一二三，一二三一二三。
> 安大娘走後,兩個孩子果然聽話不出,在屋裏講了幾個故事,又捉了半天迷藏,後來拿些小碗小筷,裝煮飯。   

生成的广东话音频

https://github.com/PaddlePaddle/PaddleSpeech/assets/32130780/ab110986-dec3-4d32-8ff9-51bef03d2ad3

版本
```
paddle-bfloat==0.1.7
paddle2onnx==1.1.0
paddleaudio==1.1.0
paddlefsl==1.1.0
paddlehub==2.4.0
paddlenlp==2.5.2
paddlepaddle==2.5.2
paddlesde==0.2.5
paddleslim==2.4.1
paddlespeech==1.4.1
paddlespeech-feat==0.1.0
```

**Comment by garywill** [comment]
@WongLaw @JiehangXie

**Comment by JiehangXie** [comment]
这是自动回复邮件。
 
您好，您的邮件已到达我的邮箱，我将尽快查阅并回复，谢谢。Hello, I have reveived your email. But due to the time differences and my busy work, perhaps I will reply to you in 24 to 48 hours.

**Comment by EthanChin-ai** [comment]
您好，请问这个问题找到解决方案了吗？
